samples/dect/dect_phy/dect_shell/src/dect/dect_phy_shell.c

```diff
@@ -742,6 +742,11 @@ static const char dect_phy_ping_cmd_usage_str[] =
 	"      --c_tx_pwr <int>,      TX power (dBm),\n"
 	"                             [-40,-30,-20,-16,-12,-8,-4,0,4,7,10,13,16,19,21,23].\n"
 	"                             Default: from common tx settings.\n"
+	"      --c_tx_lbt_period <cnt>,  Listen Before Talk (LBT) period (symbol count).\n"
+	"                                Zero value disables LBT (default).\n"
+	"                                Valid range for symbol count: 2-110.\n"
```

Comment:
PR description says range is 1-110. Which is correct?

Reply:
2, I'll fix commit/pr description.